src/generic/stage1/json_structural_indexer.h

```diff
@@ -195,4 +197,91 @@ really_inline error_code json_structural_indexer::finish(dom_parser_implementati
   return checker.errors();
 }
 
+/**
```

Comment:
Not really a comment, but it seems that this particular function keeps on moving!!!

Reply:
Um. Thanks for the comment. I don't know why these got added here. I bet it's a merge thing. I just submitted an update that removes them. find_next_document_index.h is still their home.